packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { string, element, oneOf, shape, arrayOf } from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const ltrRtl = (ltrValue, rtlValue) => ({ dir }) =>
+  dir === 'ltr' ? ltrValue : rtlValue;
+
+const topicTagParent = `
+  word-break: break-word;
+  & a {
+    display: flex;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
+    text-decoration: none;
+    color: ${C_EBON};
+
+    &:hover, &:focus {
+      text-decoration: underline;
+    }
+    &:visited {
+      color: ${C_METAL};
+    }
+  }
+
+  & a > span {
+    background-color: ${C_LUNAR};
+    padding: 0.375rem 0.4375rem;
```

Comment:
Where have these arbitrary values come from and are they necessary? From what I can see this is 6px and 7px. If you're referring to Zeplin and units are not multiples of 4, then it is often inaccurate and you can apply your interpretation using GEL_SPACING values. Where spacing genuinely needs to be 'off', create a descriptive constant that holds this value.

Reply:
I spoke with Yan about these spacings, and confirmed that the spacings are 6px and 7px respectively, I can fire up a PR that would introduce these as constants in gel-foundations 👍

Reply:
Sorry, I should have been clearer! I mean a constant local to the file (rather than in `gel-foundations`). For example, see `MIN_TAP_HEIGHT` in the following excerpt:

```js
//...

import {
  GEL_MARGIN_BELOW_400PX,
  GEL_MARGIN_ABOVE_400PX,
  GEL_SPACING_DBL,
  GEL_SPACING,
} from '@bbc/gel-foundations/spacings';

const MIN_TAP_HEIGHT = '2.75rem';

const COMMON_HEADING_STYLES = `
  color: ${C_WHITE};
  margin-top: 0;
  margin-bottom: 0;
`;

const Wrapper = styled.div`
  ${({ service }) => getSansRegular(service)}
  background-color: ${C_CONSENT_BACKGROUND};
  max-height: 100vh;

// ...
```

Reply:
Ah! My mistake, that makes a lot of sense, I'll implement that now 👍 Thanks!